fe/java-udf/pom.xml

```diff
@@ -179,6 +168,12 @@ under the License.
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
Why move this to the last?
If there is special reason, add comment in code.

Reply:
conflict with hudi-bundle